nielslange/woo-test-environment
# Only create Cart & Checkout pages if the WooCommerce Blocks is active

The goal of this PR is to allow the command execution to finish even when there's no sidebar.

Fixes https://github.com/nielslange/woo-test-environment/issues/3

### Testing
1. On a fresh site, run the command: `wp woo-test-environment setup`
2. Go to `wp-admin/edit.php?post_type=page` and check the `Cart` and `Checkout` pages don't exist.
5. Run `wp woo-test-environment setup --version=7.9.0`
6. Go to `wp-admin/edit.php?post_type=page` and check the `Cart` and `Checkout` pages are created and work.
